lib/images.sh

```diff
@@ -6,8 +6,8 @@ if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
   export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_20.04_NODE_IMAGE_K8S_v1.18.8.qcow2}
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.18.8/}
 elif [[ "${IMAGE_OS}" == "FCOS" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-30.20191014.0-openstack.x86_64.qcow2}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20191014.0/x86_64/}
+  export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-32.20200923.2.0-openstack.x86_64.qcow2.xz}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20200923.2.0/x86_64/fedora-coreos-32.20200923.2.0-openstack.x86_64.qcow2.xz}
```

Comment:
That would not work, the name is concatenated to the location.

Reply:
Yes exactly, but if name is removed from location, link is just [useless](https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20200923.2.0/x86_64/) but [location](https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/32.20200923.2.0/x86_64/fedora-coreos-32.20200923.2.0-openstack.x86_64.qcow2.xz) with name concatenated it straight goes to download. Should I still remove name concatenated from location?

Reply:
The location of the image is [here](https://builds.coreos.fedoraproject.org/streams/testing.json) under `artifacts-->openstack`.

Reply:
yes you should remove it. when fetching the image, the two variables are concatenated

Reply:
Done